iocore/net/UnixNetAccept.cc

```diff
@@ -116,7 +117,10 @@ net_accept(NetAccept *na, void *ep, bool blockable)
 NetAccept *
 getNetAccept(int ID)
 {
-  return naVec.at(ID);
+  SCOPED_MUTEX_LOCK(lock, naVecMutex, this_ethread());
+  auto ret = naVec.at(ID);
+  MUTEX_RELEASE(lock);
```

Comment:
We don't need to release lock like this?

Reply:
It doesn't need to be *explicitly* released - that's the point of `SCOPED_MUTEX_LOCK`. It creates a local variable (in this case `lock`, the first argument) so that when that local goes out of scope, the lock is released. Then it always works even if there are multiple returns or even exceptions.